docs/packages/package_main.rst

```diff
@@ -3,46 +3,26 @@
 Mesa Packages 
 ==============
 
-*Supporting MESA Agent Based Modeling with Crowd Sourced Packages*  
-------------------------------------------------------------------
+		*Bottom up models are virtual laboratories where controlled experiments distinguish noise from signal in the systems organization....
+		This approach may change our whole notion of scientific theory, which until now has been based on the theories of physics. Theories of
+		complex systems may never be reducible to simple analytical equations, but are more likely to be sets of conceptually simple mechanisms
+		(e.g. Darwinian natural selection) that produce different dynamics and outcomes in different contexts.*
 
-	*Bottom up models are virtual laboratories where controlled experiments distinguish noise from signal in the systems organization....
-	This approach may change our whole notion of scientific theory, which until now has been based on the theories of physics. Theories of
-	complex systems may never be reducible to simple analytical equations, but are more likely to be sets of conceptually simple mechanisms
-	(e.g. Darwinian natural selection) that produce different dynamics and outcomes in different contexts.*
+		Volker Grimm et al, "Pattern Oriented Modeling of Agent Based Complex Systems: Lessons from Ecology" Science, New Series, Vol. 310, No. 5750
+		(Nov. 11, 2005), pp. 987-991
 
-	Volker Grimm et al, "Pattern Oriented Modeling of Agent Based Complex Systems: Lessons from Ecology" Science, New Series, Vol. 310, No. 5750
-	(Nov. 11, 2005), pp. 987-991
 
 
-Mesa Packages Purpose: 
-----------------------
+The Mesa core functionality is just a subset of what we believe researchers creating Agent Based Models (ABMs) will use. We designed Mesa to be extensible, so that individuals from various domains can build and maintain their own Mesa packages in pursuit of "unifying algorithmic theories of the relation between adaptive behavior and system complexity."
```

Comment:
@tpike3 ... re: "“unifying algorithmic theories of the relation between adaptive behavior and system complexity.”" ... why is that in quotes?